backend/root/utils/mixins.py

```diff
@@ -173,3 +179,105 @@ class Meta:
     def save(self, *args: Any, **kwargs: Any) -> None:
         self.full_clean()
         super().save(*args, **kwargs)
+
+
+class CustomBaseSerializer(serializers.ModelSerializer):
+    created_by = serializers.SerializerMethodField(method_name='get_created_by', read_only=True)
+    updated_by = serializers.SerializerMethodField(method_name='get_updated_by', read_only=True)
+    """
+        Base serializer, sets version fields to read_only
+        Adds validation errors from models clean
+        Context of request needs to be passed
+    """
```

Comment:
docstrings må være øverst tror jeg
```suggestion
    """
        Base serializer, sets version fields to read_only
        Adds validation errors from models clean
        Context of request needs to be passed
    """
    created_by = serializers.SerializerMethodField(method_name='get_created_by', read_only=True)
    updated_by = serializers.SerializerMethodField(method_name='get_updated_by', read_only=True)
```

Reply:
Uff ja oversight det der